client/Cargo.toml

```diff
@@ -39,8 +39,8 @@ sp-application-crypto = { version = "4.0.0-dev", git = "https://github.com/parit
 sp-core = { version = "4.0.0-dev", git = "https://github.com/paritytech/substrate.git", branch = "master" }
 
 #local dependencies
-substratee-node-primitives = { path = "../primitives/node" }
-substratee-worker-primitives = { path = "../primitives/worker" }
-substratee-api-client-extensions = { path = "../primitives/api-client-extensions" }
-substratee-stf = { path = "../stf" }
-substratee-worker-api = { path = "../worker/rpc/client" }
\ No newline at end of file
+itp-teerex = { path = "../primitives/node" }
```

Comment:
I think we need to make sure the folder structure reflects the new crate names. please rename the directories accordingly. otherwise it will be hard to find stuff